src/taco-cli/cli/run.ts

```diff
@@ -186,7 +190,7 @@ class Run extends commands.TacoCommandBase {
             name: "local",
             run: Run.local,
             canHandleArgs(commandData: commands.ICommandData): boolean {
-                return !!commandData.options["local"];
+                return !!commandData.options["local"] || !!commandData.options["livereload"] || !!commandData.options["devicesync"];
```

Comment:
This might not have the desired effect: With the code this way, if I say `taco run ios` it will run it remotely if I have configured a remote, but `taco run ios --livereload` will try and run it locally. That seems a bit confusing to me; might be better to error out if livereload or devicesync are specified and we would otherwise run remotely?